orgs-using-d.dd

```diff
@@ -122,6 +122,13 @@ $(DIVC orgs-using-d center,
             $(FA_HIRING www.weka.io)
         )
     )
+    $(DORG Ecratum, http://www.ecratum.com, ecratum.png,
+        Supplier Relationship Management tool for SMBs,
+        Using dlang to run some of our core apps (support app, public API), 
```

Comment:
You can see the output [here](http://dtest.thecybershadow.net/artifact/website-2b05bdb6c33e43dd38cd39ddfa8f59c5febbbae1-a00a660c4d91e1a88fd10b16a4ec916b/web/orgs-using-d.html) or preview it with:

```
make -f posix.mak html
```